pillarbox-demo-shared/src/main/java/ch/srgssr/pillarbox/demo/shared/ui/settings/AppSettings.kt

```diff
@@ -27,9 +27,9 @@ class AppSettings(
      * @property size the [TextUnit].
      */
     enum class TextSize(val size: TextUnit) {
```

Comment:
We should have bigger text size for tv, maybe use resource id for the sizes?

Reply:
I've replaced hard coded text sizes with `TextStyle` from `MaterialTheme`. This way we have a consistent display on mobile or TV.